api/scripts/restore-dump.sh

```diff
@@ -1,2 +0,0 @@
-lz4cat meta-closure.sql.lz4 | psql -U postgres -h localhost -p 5433 measurements -f sample-dump.sql
-
```

Comment:
What does refresh this script?

Reply:
I don't understand the question.

Reply:
Neither do I, it was phrased incorrectly. I mean to ask:

> What does replace this script?

Reply:
No idea, probably nothing.